apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -364,10 +365,32 @@ const Ticket = new GQLListSchema('Ticket', {
         lastCommentAt: {
             schemaDoc: 'Last comment time in ticket',
             type: DateTimeUtc,
+            access: {
+                create: userIsAdmin,
+                read: true,
+                update: userIsAdmin,
+                delete: false,
+            },
```

Comment:
It would be great to add such access for other fields that should not be updated by the user. Not in this pr

Reply:
https://jira.in.doma.ai/browse/DOMA-8830
Create task for this